app/controllers/attendances_controller.rb

```diff
@@ -13,7 +13,7 @@ def destroy
   private
 
     def attendance
-      current_user.attendances.find(params[:id])
+      current_user.student_team.attendances.find(params[:id])
```

Comment:
😂 well I've never touched this area of the app. But if that's true, and the controller does not have **any** touching point with the rest of the app, than it should probably be removed altogether.

Can you maybe explore this a bit deeper if you can really confirm it's not being used?